### PR TITLE
[python-frontend] Add support for mod.cls.method calls

### LINKED ISSUE #1901: [python-frontend] Import the whole module
ESBMC should handle imports like this:

```python
import my_module

n:int = my_module.my_class.my_classmethod()

assert n == 0
```
from a module with a class definition:
```python
class my_class:
    @classmethod
    def my_classmethod(cls) -> int:
        return 0
```

### PR DESCRIPTION
**Pull request by ivomachado**
To allow chained call when there is an "import module" and a call to a class method with "module.class.classmethod()".

### CODE REVIEW

File: src/python-frontend/json_utils.h [hunk 1]
```diff
@@ -44,6 +44,25 @@ bool is_class(const std::string &name, const JsonType &ast_json)
...
+        std::ifstream imported_file(module_path.str());
+        if (!imported_file.is_open())
+          return false;
+
+        JsonType imported_module_json;
+        imported_file >> imported_module_json;
+
+        if (is_class(name, imported_module_json))
+          return true;
+      }
```
Comment (brcfarias) [inline src/python-frontend/json_utils.h]: Please define a lambda for this, and call it in both blocks.

File: src/python-frontend/python_converter.cpp
Comment (brcfarias) [inline src/python-frontend/python_converter.cpp:1]: Run `clang-format-11 -i src/python-frontend/python_converter.cpp` to fix the indentation.

File: src/python-frontend/python_converter.cpp [hunk 2]
```diff
@@ -545,7 +545,14 @@ function_id python_converter::build_function_id(const nlohmann::json &element)
   {
     is_member_function_call = true;
     func_name = func_json["attr"];
-    obj_name = func_json["value"]["id"];
+    if (func_json["value"]["_type"] == "Attribute")
+    {
```
Comment (brcfarias) [inline src/python-frontend/python_converter.cpp]: We try to avoid using curly braces for single-line if statements. &#128578;

**Review by brcfarias — CHANGES_REQUESTED** [review]

File: regression/python/import-whole-module/main.py
Comment (brcfarias) [inline regression/python/import-whole-module/main.py:1]: `regression/python/import/main.py` contains an import in the same format (whole module).
The diference is that a class method is being invoked here. Could you please merge them by moving the class definition and class method call there?

**Review by brcfarias — CHANGES_REQUESTED** [review]

**Comment by brcfarias** [comment]
Fixes #1901.

File: src/python-frontend/json_utils.h (hunk 1 above)
Comment (ivomachado) [reply src/python-frontend/json_utils.h]: Done

File: src/python-frontend/python_converter.cpp
Comment (ivomachado) [reply src/python-frontend/python_converter.cpp:1]: Done

File: src/python-frontend/python_converter.cpp (hunk 2 above)
Comment (ivomachado) [reply src/python-frontend/python_converter.cpp]: Done

File: regression/python/import-whole-module/main.py
Comment (ivomachado) [reply regression/python/import-whole-module/main.py:1]: Done

**Review by lucasccordeiro — APPROVED** [review]

**Comment by lucasccordeiro** [comment]
Thanks for submitting this PR, @ivomachado.

**Pull request merged**